release-highlights/release-highlights-2021.05/release-highlights.md

```diff
@@ -0,0 +1,74 @@
+
+### [Inline-compact Clipboard copy variant](https://www.patternfly.org/v4/components/clipboard-copy#inline-compact)
+
+![inline compact clipboard copy](./img/inline-clipboard-copy.png)
+
+Adds a thin variant to Clipboard copy for using inline with text or anywhere a lighter weight treatment is desired.
+
+### [Code block component](https://www.patternfly.org/v4/components/code-block)
+
+![code block component](./img/code-block.png)
+
+Introduces a read-only code block component that is lighter weight than the full code editor for displaying a snippet of code with optional actions. The ability to show or hide lines is available for long code snippets.
+
+### [Optional item counts in a Select menu](https://www.patternfly.org/v4/components/select#checkbox-input-with-counts)
+
+![item counts in a select menu](./img/select-item-counts.png)
+
+Adds optional right-aligned item counts to a Select menu. This is useful for faceted filtering use cases where you many want to report the number of items matching each specific value.
+
+### [Add validated states to Select](https://www.patternfly.org/v4/components/select#validated)
+
+![validated select menu](./img/validated-select.png)
+
+Allows showing validation states (e.g. error, warning, success) for a Select component. Visual appearance is equivalent to the same states displayed for text inputs.
+
+### [Date and time range picker demo](https://www.patternfly.org/v4/components/date-picker/react-demos#date-and-time-range-picker)
+
+![date and time range demo](./img/date-time-picker.png)
+
+Adds a React demo to show how the Date picker and Time picker components can be used together to return a data and time range.
+
+### [Slider updated API to match standard range input](https://www.patternfly.org/v4/components/slider)
+
+The slider component has been updated to mimic the API of a standard HTML range input with additional custom features.
+
+See the [latest release notes](https://www.patternfly.org/v4/developer-resources/release-notes) for a more detailed list of changes.
+
+-----------------------------------------------------------------------------
+
+## What we’re working on...
+
+### 2021.06 (May 14)
+
+* [Log viewer component](https://github.com/patternfly/patternfly-react/issues/5341) - Introduce a native log viewer native beta component to display real-time streamed logs or static log data.
+
+* [Update toggle group styling](https://github.com/patternfly/patternfly/issues/3951) - Update selected color for a set of toggle buttons to use a lighter blue.
+
+* [New List variants](https://github.com/patternfly/patternfly-react/issues/5336) - Add new variants of the List component to include horizontal separators, icons and 'pf-m-plain' modifier.
+
+* [Composable dropdown menus](https://github.com/patternfly/patternfly-react/issues/5073) - Create a demo that shows how the Menu and Toggle Button components can be combined to create custom dropdown elements.
```

Comment:
Is the capitalization correct here?

Reply:
Actually should be "Menu toggle" and not "Toggle button." Fixed.